hbs-helpers.js

```diff
@@ -1,78 +1,86 @@
 "use strict";
 
 const HBSHelpers = {
-  init(hbs) {
+  breachDataClasses(dataClasses) {
+    if (dataClasses.constructor === Array) {
+      return dataClasses.join(", ");
+    } else {
+      return "";
```

Comment:
Still curious why this is needed. Are we seeing breaches without classes? Or is classes not an array?
If not an array, should this just return the raw `dataClasses`?

Reply:
I couldn't reproduce the issue on my local, so this is just some defensive coding.

Returning the raw data classes string is probably preferred here, but I've got more commits based on this one so I'd like to merge this, send the expanded PR, and make that fix in the other PR.

Reply:
OK, I had a few minutes to dig deeper, but I don't think I'm any closer to understanding this...

```js
const got = require("got");

async function main() {
  const breaches = await got("https://haveibeenpwned.com/api/v2/breaches", {json:true});

  breaches.body.forEach(b => console.log(`${b.Title} ==> ${b.DataClasses} (${Array.isArray(b.DataClasses)}::${b.DataClasses.length})`));
}

main();
```

Running that script will download all the latest breach info from HIBP (although production) endpoint and display the Title, DataClasses, and number of breached data classes, a la:

```sh
000webhost ==> Email addresses,IP addresses,Names,Passwords (true::4)
126 ==> Email addresses,Passwords (true::2)
17 ==> Device information,Email addresses,IP addresses,Passwords,Usernames (true::5)
17173 ==> Email addresses,Passwords,Usernames (true::3)
2,844 Separate Data Breaches ==> Email addresses,Passwords (true::2)
2fast4u ==> Email addresses,Passwords,Usernames (true::3)
7k7k ==> Email addresses,Passwords,Usernames (true::3)
...
Xbox-Scene ==> Email addresses,IP addresses,Passwords,Usernames (true::4)
xHamster ==>  (true::0)
XPG ==> Email addresses,IP addresses,Passwords,Usernames (true::4)
...
```

I couldn't find any records where `b.DataClasses` was not an array, although I did notice that the only breach with zero DataClasses was xHamster: https://fx-breach-alerts.herokuapp.com/?breach=xHamster